floodviz/hydrograph_utils.py

```diff
@@ -2,12 +2,12 @@
 import time
 import requests
 
-
 def parse_hydrodata(jdata):
     
     """ 
     Parses json Hydrodata from NWIS webservice
-    and formats for NVD3 charting library.
+    and formats for NVD3 charting library. Upon failure, this will
+    return empty data list.
```

Comment:
I think that's fine then. We can always reconsider later if this function is used for something else.